picamera2/previews/drm_preview.py

```diff
@@ -110,7 +110,7 @@ def set_overlay(self, overlay):
         else:
             h, w, channels = overlay.shape
             # Should I be recycling these instead of making new ones all the time?
-            new_fb = pykms.DumbFramebuffer(self.card, w, h, "AB24")
+            new_fb = pykms.DumbFramebuffer(self.card, w, h, "XR24")
```

Comment:
Might need to think about this... presumably an overlay has to have alpha to be useful?

Reply:
Indeed. Again I was being a bit over-enthusiastic as I hadn't worked out the magic runes of `picam2.create_preview_configuration({"format": "XRGB8888",`, so was seeing bad formats kicking around and changed them.

Reply:
Understood! I'll probably leave this one as it was. If it fails, applications either shouldn't call it, or stick it in a try/except.